sdk/communication/Shared/src/HMACAuthenticationPolicy.cs

```diff
@@ -45,7 +45,7 @@ private static string CreateContentHash(HttpMessage message)
                 message.Request.Content?.WriteTo(contentHashStream, message.CancellationToken);
             }
 
-            return Convert.ToBase64String(alg.Hash);
+            return Convert.ToBase64String(alg.Hash!);
```

Comment:
What does this do?

Reply:
`alg.Hash` is marked as nullable while `Convert.ToBase64String` argument is marked as non-nullable. This tells C# compiler that we are positive that `alg.Hash` won't be null in this particular case.